zookeeper-server/src/test/java/org/apache/zookeeper/server/quorum/DIFFSyncConsistencyTest.java

```diff
@@ -49,7 +49,7 @@ public class DIFFSyncConsistencyTest extends QuorumPeerTestBase {
     private MainThread[] mt = new MainThread[SERVER_COUNT];
 
     @Test
-    @Timeout(value = 120)
+    @Timeout(value = 20)
```

Comment:
What's the reason for this timeout change? The tests seems to pass through this change.

Reply:
I believe this test was unstable, and when it hung, it was annoying that it hung for so long. We don't need to commit this change, though. 
```suggestion
    @Timeout(value = 120)
```